ports/windows/mpconfigport.h

```diff
@@ -201,6 +204,14 @@ extern const struct _mp_obj_module_t mp_module_time;
 
 #define MICROPY_MPHALPORT_H         "windows_mphal.h"
 
+#define MICROPY_EVENT_POLL_HOOK \
```

Comment:
Shouldn't this be guarded by `#if MICROPY_ENABLE_SCHEDULER`, to not change what mp_hal_delay_ms does in case scheduler is off?

Reply:
MICROPY_EVENT_POLL_HOOK is also needed for a uselect branch I'm about to push up, but yes it probably is cleaner to guard this here, even if it's not switched as such in the unix port. Perhaps I should add a similar compile guard there too...